### PR TITLE
feat(meshservice): add namespace column to listing

### LINKED ISSUE #2560: MeshService: add namespace column to MeshService listing
### Description

In https://github.com/kumahq/kuma-gui/pull/2546 we added a new listing+summary panel for our new MeshServices

You can see the listing here:

https://deploy-preview-2546--kuma-gui.netlify.app/gui/meshes/default/services/mesh-services?page=1&size=50#KUMA_ENVIRONMENT=kubernetes

Screenshot 2024-05-15 at 15 34 12

(Note, in order to see the namespaces during development you must add a `KUMA_ENVIRONMENT=kubernetes` to your browser cookies. The above `#KUMA_ENVIRONMENT=kubernetes` automatically does that for you)

In other areas (for example Dataplanes and ZoneIngress/Egress) we split off the namespace from the name and put it in a separate column:

Screenshot 2024-05-15 at 15 36 08

We should also add a Namespace column for MeshService

---

Handy prior work / useful links:

- https://github.com/kumahq/kuma-gui/blob/9dfdcfac45466501205dd8b5bb1bbc8d60ebe9b7/src/app/data-planes/data/index.ts#L229-L231
- https://github.com/kumahq/kuma-gui/blob/9dfdcfac45466501205dd8b5bb1bbc8d60ebe9b7/src/test-support/mocks/src/meshes/_/meshservices.ts#L36-L39

### PR DESCRIPTION
**Pull request by schogges**
This change splits the name in the `MeshService` listing into the `name` and the `namespace`. A new column is introduced for the `namespace`.

Closes #2560

### CODE REVIEW

File: features/mesh/services/mesh-services/Index.feature [hunk 1]
```diff
@@ -29,8 +29,8 @@ Feature: mesh / services / mesh-services / index
         When I visit the "<URL>" URL
         Then the "$button-group" element exists
         And I click the "$item a" element
-        Then the URL contains "monitor-proxy-0.kuma-demo"
-        And the URL doesn't contain "monitor-proxy-0.kuma-demo/overview"
+        Then the URL contains "monitor-proxy-0"
+        And the URL doesn't contain "monitor-proxy-0/overview"
```
Comment (johncowen) [inline features/mesh/services/mesh-services/Index.feature:33]: We still need to use the full `name.namespace` in the URL to keep it unique.

Elsewhere we add an `id` property for the resource/object containing the `name.namespace` and we use that as the unique thing for links/URLs etc, and then use the `name` for the thing we render to the page.

https://github.com/kumahq/kuma-gui/blob/9dfdcfac45466501205dd8b5bb1bbc8d60ebe9b7/src/app/zone-ingresses/data/index.ts#L75

https://github.com/kumahq/kuma-gui/blob/9dfdcfac45466501205dd8b5bb1bbc8d60ebe9b7/src/app/zone-ingresses/views/ZoneIngressListView.vue#L67

**Review by johncowen — COMMENTED** [review]
👍 Looks good up to now, little more to do I think, I left an inline to explain

File: features/mesh/services/mesh-services/Index.feature (hunk 1 above)
Comment (schogges) [reply features/mesh/services/mesh-services/Index.feature:33]: Ah thank you, that makes sense 🙂

**Review by johncowen — APPROVED** [review]
Looks good, thanks!

**Pull request merged**